### PR TITLE
Accept ed25519 and other common SSH Keys

### PR DESCRIPTION
**Pull request by bobbykdhan**
#199 Here you go @jabbate19 

Updated the regex expression to accept other common ssh keys besides RSA

### CODE REVIEW

**Comment by jabbate19** [comment]
Fixes #199

**Review by jabbate19 — CHANGES_REQUESTED** [review]
This only works for letter only types, but there are other big ones (ed25519)

File: proxstar/static/js/script.js [hunk 1]
```diff
@@ -223,7 +223,7 @@ $("#create-vm").click(function(){
     const max_cpu = $(this).data('max_cpu');
     const max_mem = $(this).data('max_mem');
     const max_disk = $(this).data('max_disk');
-    var ssh_regex = new RegExp("ssh-rsa AAAA[0-9A-Za-z+/]+[=]{0,3}( [^@]+@[^@]+)?")
+    var ssh_regex = new RegExp("ssh-[a-z]+ AAAA[0-9A-Za-z+/]+[=]{0,3}( [^@]+@[^@]+)?");;
```
Comment (jabbate19) [inline proxstar/static/js/script.js:226]: Two semicolon?

**Review by jabbate19 — CHANGES_REQUESTED** [review]
image
Does not fix ed25519 or other algorithms with numbers in the spec

File: proxstar/static/js/script.js (hunk 1 above)
Comment (jabbate19) [reply proxstar/static/js/script.js:226]: Also, why are you adding the `AAAA` to the regex? This is not a guarantee to be there iirc

Comment (bobbykdhan) [reply proxstar/static/js/script.js]: it was in the original

**Review by jabbate19 — APPROVED** [review]
Cool

File: proxstar/static/js/script.js [hunk 2]
```diff
@@ -223,7 +223,7 @@ $("#create-vm").click(function(){
     const max_cpu = $(this).data('max_cpu');
     const max_mem = $(this).data('max_mem');
     const max_disk = $(this).data('max_disk');
-    var ssh_regex = new RegExp("ssh-rsa AAAA[0-9A-Za-z+/]+[=]{0,3}( [^@]+@[^@]+)?")
+    var ssh_regex = new RegExp("ssh-[a-z0-9]+ AAAA[0-9A-Za-z+/]+[=]{0,3}( [^@]+@[^@]+)?");
```
Comment (galenguyer) [inline proxstar/static/js/script.js:226]: this won't match security keys created with a hardware auth key (yubikey, solokey, etc)

Comment (bobbykdhan) [reply proxstar/static/js/script.js:226]: who asked?

Comment (bobbykdhan) [reply proxstar/static/js/script.js:226]: image

**Pull request merged**